### PR TITLE
4382 Route and UI

### PR DESCRIPTION
**Pull request by matthopson**
Screen Shot 2020-02-24 at 2 09 42 PM

### CODE REVIEW

File: web-client/src/presenter/computeds/caseInventoryHelper.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { state } from 'cerebral';
+
+export const caseInventoryHelper = (get, applicationContext) => {
```
Comment (rachaelparris) [inline web-client/src/presenter/computeds/caseInventoryHelper.js]: Sorry, this is a little nitpicky (feel free to tell me no 😄) but for clarity, could we add "Report" to all these names? `caseInventoryReport`?

Comment (matthopson) [reply web-client/src/presenter/computeds/caseInventoryHelper.js]: Yup!

File: web-client/src/views/CaseInventoryReport/CaseInventoryReport.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            </div>
+            <div className="grid-col-3">
+              <BindedSelect
+                ariaDescribedBy="case-deadlines-tab filterHeading"
+                ariaLabel="judge"
+                bind="screenMetadata.caseDeadlinesFilter.judge"
+                className="select-left"
+                id="judgeFilter"
+                name="judge"
+                placeHolder="- Judge -"
```
Comment (rachaelparris) [inline web-client/src/views/CaseInventoryReport/CaseInventoryReport.jsx]: There are a few 'judge' instances here that I think should be 'status'.

**Review by rachaelparris — APPROVED** [review]
One more small issue, but otherwise 👍

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**